.env.example

```
@@ -93,16 +93,25 @@ OLLAMA_MODEL=atlas-extract:latest
+# Defaults:
+#   CRON_PRUNE_SCHEDULE   03:40 UTC daily (just after the documents snapshot)
+#   CRON_STATUS_SCHEDULE  00:05 UTC daily (transitions trips planned→active,
+#                                          active→completed by date)
+# Set CRON_TZ to an IANA zone (e.g. Europe/Berlin) to interpret schedules
+# in local time instead of UTC.
+#
+# Upgrading from a pre-pg-boss Atlas: pg-boss's cron parser also accepts
+# the 6-field croner format (`sec min hour ...`), so a legacy value like
+# `0 40 3 * * *` keeps working unchanged. The 5-field form is preferred
+# going forward.
+CRON_PRUNE_SCHEDULE=40 3 * * *
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)